doc/ranger.pod

```diff
@@ -885,18 +885,21 @@ if you delete multiple files at once.
 
 =item date_format [string]
 
-The date format to be displayed in the interface. It defaults to "%x" which is the locale
-appropriate date format. See python's I<strftime> docs for valid format strings.
-To avoid ranger trying to resolve the format strings as macros, they need to be
-escaped by doubling up the "%" symbols. e.g. B<%Y-%m-%d> must be set as B<%%Y-%%m-%%d>.
+=item time_format [string]
 
 =item date_and_time_format [string]
 
-Format used when both date and time are displayed, as in the statusbar.
-It defaults to "date_format time_format".
+The formats to be used in the interface when date, time,
+or both date and time (as in the statusbar) are displayed.
+
+The defaults are "%%x" for date_format, "%%X" for time format,
+and "date_format time_format" for date_and_time_format.
 See python's I<strftime> docs for valid format strings.
+
 To avoid ranger trying to resolve the format strings as macros, they need to be
 escaped by doubling up the "%" symbols. e.g. B<%x %H:%M> must be set as B<%%x %%H:%%M>.
+This is also a good habit since those strings could be interpreted as python's 
```

Comment:
I don't think this makes sense. That would only matter if we used them as format strings but I don't think we do? What I was getting at is that `%x` doesn't actually need the doubling up, because it's not a valid macro, though it doesn't hurt either, but `%%d` *does* because it *is* a valid macro.

Reply:
> What order are settings loaded now?

They're sorted by name so `date_and_time_format` gets loaded first, even though `date_format` is declared just above it.

> I'm not sure the %s should still be escaped at this point. Did you test with some format string that has something that conflicts with a macro, %d or something?

I think I got stuck with the loading order and forgot to test those :/
I did now and they're not working when escaped, even with `%d`, so I'll have to unescape them.

> I don't think this makes sense. That would only matter if we used them as format strings but I don't think we do?

Because I'm using the _format()_ function to build the `date_and_time_format` default value and you previously mentioned them as _formatting characters_, I got the two concepts mixed-up.
Sorry about that, I'll fix it.